shell/platform/embedder/tests/embedder_unittests.cc

```diff
@@ -552,6 +552,7 @@ TEST_F(EmbedderTest,
           FlutterBackingStore backing_store = *layers[0]->backing_store;
           backing_store.type = kFlutterBackingStoreTypeSoftware;
           backing_store.did_update = true;
+          backing_store.is_cacheable = true;
```

Comment:
Due to the earlier discussion on breaking changes, these test cases may not change.